indexer/services/comlink/src/request-helpers/request-transformer.ts

```diff
@@ -307,6 +309,20 @@ export function pnlTicksToResponseObject(
   };
 }
 
+export function SubaccountInfoToTraderSearchResponse(
```

Comment:
```suggestion
export function subaccountInfoToTraderSearchResponse(
```
lowercase, because it's a function